### PR TITLE
feat: add param to VCFAnnotator to optionally require validation checks

### LINKED ISSUE #343: Allow reference sequence validation to be toggled for VCF annotation
Currently, kwargs can be passed into `_from_gnomad` to prevent validation of reference sequences (default `require_validation=True`) but this is not exposed to `VCFAnnotator.annotate()`. It would be great if we could pass in kwargs from VCFAnnotator to `_from_gnomad` so we can disable reference genome validation. Decreasing the strictness allows users to increase the VRS IDs sent back to them when annotating VCFs.

### PR DESCRIPTION
**Pull request by korikuzma**
close #343 

* Adds `require_validation` parameter. If `True`, validation checks must pass in order to return a VRS object. If validation checks fail, a ValueError will be raised. If `False`, then the VRS object will be returned even if validation checks fail.

### CODE REVIEW

**Comment by korikuzma** [comment]
@quinnwai  I don't have permissions to change your permissions so you can be added as a reviewer.

File: src/ga4gh/vrs/extras/vcf_annotation.py [hunk 1]
```diff
@@ -102,10 +102,17 @@ class SeqRepoProxyType(str, Enum):
...
+    is_flag=True,
+    default=True,
+    show_default=True,
+    help="Require validation checks to pass in order to return a VRS object"
+)
 def annotate_click(  # pylint: disable=too-many-arguments
     vcf_in: str, vcf_out: Optional[str], vrs_pickle_out: Optional[str],
     vrs_attributes: bool, seqrepo_dp_type: SeqRepoProxyType, seqrepo_root_dir: str,
-    seqrepo_base_url: str, assembly: str, skip_ref: bool
+    seqrepo_base_url: str, assembly: str, skip_ref: bool, require_validation: bool
```
Comment (quinnwai) [inline src/ga4gh/vrs/extras/vcf_annotation.py:115]: do we need to pass require_validation through to annotator.annotate below as well?

File: src/ga4gh/vrs/extras/vcf_annotation.py [hunk 2]
```diff
@@ -102,10 +102,17 @@ class SeqRepoProxyType(str, Enum):
     show_default=True,
     help="Skip VRS computation for REF alleles."
 )
+@click.option(
+    "--require_validation",
+    is_flag=True,
+    default=True,
+    show_default=True,
+    help="Require validation checks to pass in order to return a VRS object"
```
[the remark this answers is not in the document]
Comment (korikuzma) [reply src/ga4gh/vrs/extras/vcf_annotation.py:110]: Yes. Thank you!

**Review by quinnwai — APPROVED** [review]

**Comment by korikuzma** [comment]
@ga4gh/vrs-python-maintainers Is someone able to review this so I can make a pypi release for @quinnwai?

**Review by larrybabb — APPROVED** [review]
codewise this looks okay to me.

**Pull request merged**